cl/opinion_page/templates/view_opinion.html

```diff
@@ -104,16 +105,17 @@ <h4>
         {% endif %}
 
         {# Show cases this case cites #}
-        {% if top_authorities %}
+        {% if authorities_count > 0  %}
             <div id="authorities" class="sidebar-section">
-                <h3><span>Authorities ({{ cluster.authority_count }})</span></h3>
+                <h3><span>Authorities ({{ authorities_count }})</span></h3>
                 <p class="bottom">This opinion cites:</p>
                 <ul>
                     {% for authority in top_authorities %}
                         <li>
-                            <a href="{{ authority.get_absolute_url }}?{{ request.META.QUERY_STRING }}">
+                            <a href="{{ authority.absolute_url }}?{{ request.META.QUERY_STRING }}">
                                 {{ authority.caption|safe|truncatewords:10|v_wrapper }}
                             </a>
+                            ({{ authority.this_cluster_citation_count }} time{{ authority.this_cluster_citation_count|pluralize }})
```

Comment:
Same q as above. Would `gray` and/or `small` be good here?

Reply:
I think both improve the aesthetic. Fixed in 6052de5